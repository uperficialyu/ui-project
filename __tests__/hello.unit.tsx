import renderer from 'react-test-renderer';
import React from 'react';
import Button from '../lib/Button';

test('hello', () => {
  const json = renderer.create(<Button />).toJSON();
  expect(json).toMatchSnapshot();
})