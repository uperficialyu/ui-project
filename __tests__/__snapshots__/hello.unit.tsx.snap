// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello 1`] = `
<div>
  button2
</div>
`;
